website/content/docs/secrets/pki/cmpv2.mdx

```diff
@@ -0,0 +1,130 @@
+of posession of a TLS client certificate authenticates them to Vault.
+
+Authentication leverages a separate Vault authentication
+mount, within the same namespace, to validate the client provided credentials
+along with the client's ACL policy to enforce. 
+
+For proper accounting, a mount supporting CMPv2 authentication should be
+dedicated to this purpose, not shared with other workflows.  In other words,
+create a new certificate auth mount for CMPv2 even if you already have one 
+another in use for other purposes.
+
+When setting up the authentication mount for CMPv2 clients, the token type must
+be configured to return [batch tokens](/vault/docs/concepts/tokens#batch-tokens).
+Batch tokens are required to avoid an excessive amount of leases being generated
+and persisted as every CMPv2 incoming request needs to be authenticated.
+
+The path within an ACL policy must match the `cmp` path underneath the 
+PKI mount.  The path to use can be the default `cmp` path or a role based one.
+
+If using the `sign-verbatim` as a path policy, the following
+ACL policy will allow an authenticated client access the required PKI CMP path.
+```
+path “pki/cmp” {
+  capabilities=[“update”, “create”]
+}
+```
+
+For a role base path policy, this sample policy can be used
+```
+path “pki/roles/my-role-name/cmp” {
+  capabilities=[“update”, “create”]
+}
+```
+
+#### Updating the PKI mount tunable parameters
+
+Once the authentication mount has been created and configured, the authentication mount's accessor
+will need to be captured and added within the PKI mount's [delegated auth accessors](/vault/api-docs/system/mounts#delegated_auth_accessors).
+
+To get an authentication mount's accessor field, the following command can be used.
+
+```shell-session
+$ vault read -field=accessor sys/auth/auth/cert
+```
+
+For CMP to work within certain clients, a few response headers need to be explicitly allowed
+along with configuring the list of accessors the mount can delegate authentication towards.
+The following will grant the required response headers, you will need to replace the values for the `delegated-auth-accessors`
+to match your values.
+
+```shell-session
+$ vault secrets tune \
+  -allowed-response-headers="Content-Transfer-Encoding" \
+  -allowed-response-headers="Content-Length" \
+  -allowed-response-headers="WWW-Authenticate" \
+  -delegated-auth-accessors="auth_cert_4088ac2d" \
+  pki
+```
+
+#### Enabling CMPv2
+
+Enabling CMP is a matter of writing to the `config/cmp` endpoint, to set it
+enabled and configure default issuance policy and authentication.
+
+```shell-session
+vault write pki/config/cmp -<<EOC
+{
+  "enabled": true,
+  "authenticators": {
+    "cert": {
+      "accessor": "auth_cert_4088ac2d"
+    }
+  }
+}
+EOC
+```
+
+Of course, substituting your own role and accessor values.  After this, the
+CMP endpoints will be available to handle client requests, authenticated with the 
+previously configured Cert Auth method.
```

Comment:
```suggestion
```
The fact that we're enabling the endpoint already suggests that it would be available, so I'm not sure these sentences are needed.